io.gitlab.hamadmarri.gamma.json

```diff
@@ -0,0 +1,62 @@
+{
+	"app-id": "io.gitlab.hamadmarri.gamma",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.34",
+	"sdk": "org.gnome.Sdk",
+	"command": "gamma",
+	"finish-args": [
+		"--share=network",
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=ipc",
+		"--talk-name=org.freedesktop.DBus.Hello",
+		"--talk-name=org.a11y.*",
+		"--talk-name=org.freedesktop.DBus.AddMatch",
```

Comment:
Are you sure you actually need all these permissions? Sounds weird for a text editor. Neither [gedit](https://github.com/flathub/org.gnome.gedit/blob/master/org.gnome.gedit.json) or [kate](https://github.com/flathub/org.kde.kate/blob/master/org.kde.kate.json) does that.

Reply:
I am not sure about these two:
```
--talk-name=org.freedesktop.DBus.Hello
--talk-name=org.freedesktop.DBus.AddMatch
```
I can just remove them, however as suggested to see the log bus command I have seen these calls

However, the `--talk-name=org.a11y.*` when removed, I got this warning
```
dbind-WARNING **: 13:23:34.568: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```